packages/cerebral-provider-http/src/utils.js

```diff
@@ -64,3 +64,20 @@ export function convertObjectWithTemplates (obj, context) {
     return convertedObject
   }, {})
 }
+
+export function parseHeaders (rawHeaders) {
```

Comment:
For axios see https://github.com/mzabriskie/axios/blob/master/lib/helpers/parseHeaders.js.